test/test_common/test_base.h

```diff
@@ -17,18 +17,16 @@ namespace Envoy {
 //     than there were at the start of it. This is likely to fail in a few
 //     places when introduced, but we could add known test overrides for this.
 //
-// Note: nothing compute-intensive should be put in this test-class, as it will
+// Note: nothing compute-intensive should be put in this class, as it will
 // be a tax paid by every test method in the codebase.
-class TestBase : public ::testing::Test {
-public:
-  static void checkSingletonQuiescensce();
-  ~TestBase() override;
+class TestListener : public ::testing::EmptyTestEventListener {
```

Comment:
bonus points if you can think of a more descriptive name - I currently can't :-P